src/app/features/navigation-bar/navigation-bar.component.html

```diff
@@ -16,6 +16,13 @@
           [btnIcon]="this.btnCartIcon"
         ></nav-button>
       </li>
+      <li>
+        <nav-button
+          *ngIf="isUserCustomer === false"
+          (click)="openOwnerRestaurantsPage()"
+          [btnTitle]="this.btnOwnerRestaurantsTitle"
```

Comment:
I think you can just pass in the string here, why do you need to keep it in a property?